File: app/src/main/java/com/BlizzardArmory/warcraft/Statistic/Intellect.java
package com.BlizzardArmory.warcraft.Statistic;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Intellect {

    @SerializedName("base")
    @Expose
    private int base;
    @SerializedName("effective")
    @Expose
    private int effective;

    public int getBase() {
        return base;
    }

    public void setBase(int base) {
        this.base = base;
    }

    public int getEffective() {
        return effective;
    }

    public void setEffective(int effective) {
        this.effective = effective;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/statistic/Intellect.kt
package com.BlizzardArmory.model.warcraft.statistic

import com.google.gson.annotations.SerializedName


/**
 * The type Intellect.
 */
class Intellect(

        @SerializedName("base")
        var base: Int,

        @SerializedName("effective")
        var effective: Int

)